tests/tck/features/yield/parameter.feature

```diff
@@ -209,3 +209,13 @@ Feature: Parameter
       """
     Then a SyntaxError should be raised at runtime: Direct output of variable is prohibited near `$var'
     Then clear the used parameters
+
+  Scenario: expression with parameters
```

Comment:
Emm. That's not parameter but variable.

Reply:
Using the same name with @caton-hpg. He would update this scenario in the end.